src/Frontend/camFrontend/src/app/model/dispensable-result.ts

```diff
@@ -25,4 +35,13 @@ export class DispensableResult {
     looser: string; // the losing object of the results shown
     winnerTotalScore: string;
     looserTotalScore: string;
+
+    // Author: Ali
+    obj1: string;
+    obj2: string;
+    LinksObj1: Array<string>;
+    LinksObj2: Array<string>;
+    winnerSentencesObj1: Array<Sentence>;
+    looserSentencesObj2: Array<Sentence>;
```

Comment:
Is this the sentences list for object 1 and 2?
If so, please name it `obj1Sentences` and `obj2Sentences`, so that nobody gets confused with winner/looser attributes.